app/views/vendors/vendor_sales.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2>
+  All Sales:
+</h2>
+  <ul>
+
+    <% @sales.each do  |sale| %>
+       <% money = sale.amount/100 %>
+      <li>Sale Amount: $<%= money %>.00 </li>
```

Comment:
Are sale amounts always perfectly rounded? (no cents)? Because this might break otherwise.